docs/components/Combobox/Web.stories.tsx

```diff
@@ -139,7 +139,7 @@ const ComboboxClearSelection: ComponentStory<typeof Combobox> = args => {
         type="primary"
         onClick={() => setSelected(null)}
       />
-      <Combobox {...args}>
+      <Combobox {...args} multiSelect>
```

Comment:
ah whoops this was from me adding it during the screen share. I'm fine with removing it later if you are

this web story is still valid since it's about clearing selections, plus it does nothing functionally at this point. just a little thing we'll need to remember to tidy up later

Reply:
ok yaaa lets tidy in the next PR!